formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/FormstackSarService.scala

```diff
@@ -6,10 +6,12 @@ import io.circe.parser.decode
 import com.gu.identity.formstackbatonrequests.aws.SubmissionTableUpdateDate
 import io.circe.Json
 import io.circe.generic.JsonCodec
+import cats.implicits._
 
 trait FormstackSar {
   def accountFormsForGivenPage(page: Int, accountToken: FormstackAccountToken): Either[Throwable, FormstackFormsResponse]
   def formSubmissionsForGivenPage(page: Int, formId: String, minTime: SubmissionTableUpdateDate, encryptionPassword: String, accountToken: FormstackAccountToken): Either[Throwable, FormstackFormSubmissionsResponse]
+  def submissionData(submissionIdEmails: List[SubmissionIdEmail], config: PerformSarLambdaConfig): Either[Throwable, List[FormstackSubmissionQuestionAnswer]]
 }
 /* Codecs for decoding accountFormsForGivenPage response */
 @JsonCodec case class FormstackForm(id: String)
```

Comment:
would it make it easy if we removed Formstack from all these case classes? They belong to Formstak which is claer from the class and package name,

Reply:
agree - done